apps/web/src/components/commands/social/lens/command.ts

```diff
@@ -0,0 +1,35 @@
+import { CommandConfiguration } from "../../types";
+import { LensConfig } from "./Config";
+import icon from "./icon.svg";
+import { LensWidgetProps } from "./Lens";
+
+
+export const lensProfileCommand: CommandConfiguration<LensWidgetProps> = {
+  command: "lens-profile",
+  title: "Lens Profile",
+  description: "Lens statistics for a profile",
+  icon,
+
+  blockType: "inline",
```

Comment:
Is the profile really inline or should it be a block? @Kenny-Gin1 @ericzakariasson

Reply:
That is true, should probably be a block!